boards/imix/src/lowpan_frag_dummy.rs

```diff
@@ -17,13 +17,13 @@
 //! frames will prevent the test from completing successfully.
```

Comment:
On the whole, this file could probably use some more comments. It is much easier to follow now than before thanks to the changed structure shortening the file and the names of the structs better reflecting their purpose, but I still think that this example implementation would benefit from comments describing certain design decisions made specific to this test. (ex: how many rx clients are initialized, what the purpose of the alarm is and how it is used, etc.)

Reply:
Alright, sure. I'll try to update this file (and the `sixlowpan_dummy` file) with additional comments.

Reply:
One other comment on this file: It seems like it might be useful if some of the enums like TF, SAC, and DAC were defined in the sixlowpan.rs file so that any code which wants to use the sixlowpan file doesn't have to define these enums itself, and could instead just import them.

Also: I am just making single comments as I come across things while making a UDP capsule, but if I should stop doing that and compile these into a single review I can do that - not sure what people would normally do.

Reply:
So these enums are just used internally to test the different compression methods for 6LoWPAN. The `TF` enum is the only one that might be used elsewhere (and that one would be handled by the IP layer, not the 6LoWPAN layer). The rest of these are just to test the different type of source/destination addresses we might run into/need to compress.